pkg/util/clusteragent/clusteragent.go

```diff
@@ -47,6 +47,12 @@ type DCAClient struct {
 	clusterAgentAPIRequestHeaders http.Header
 }
 
+// ResetGlobalClusterAgentClient is a helper to remove the current DCAClient global
+// It is ONLY to be used for tests
+func ResetGlobalClusterAgentClient() {
```

Comment:
should not be exported then

Reply:
True, I followed blindly the Leader election  https://github.com/DataDog/datadog-agent/blob/master/pkg/util/kubernetes/apiserver/leaderelection/leaderelection.go#L72